source/braille.py

```diff
@@ -1937,7 +1937,7 @@ def func(cls):
 #: Maps old braille display driver names to new drivers that supersede old drivers.
 RENAMED_DRIVERS = {
 	"syncBraille":"hims",
-	"alvaBc6":"alva"
+	"alvaBC6":"alva"

```

Comment:
This fix works for me: if I set display to alvaBC6 and then start the try build, the ALVA display is correctly initialized. On master it is not.